hw/mcu/dialog/da1469x/include/mcu/da1469x_hal.h

```diff
@@ -21,6 +21,7 @@
 #define __MCU_DA1469X_HAL_H_
 
 #include <assert.h>
+#include "mcu/cmsis_nvic.h"
```

Comment:
cmsis_nvic.h seems to be unrelated to change

Reply:
I guess you can say it is unrelated. Looks like cmsis_nvic.h brings in mcu.h which is needed for files that include da1469x_hal.h. I will change it to mcu.h